dev-docs/get-started-with-prebid-server.md

```diff
@@ -93,6 +93,31 @@ pbjs.que.push(function() {
 });
 {% endhighlight %}
 
+Optionally, if you chose to use one of the existing Prebig.org members as your server host, you can also use the following type of configuration:
+
+{% highlight js %}
+var pbjs = pbjs || {};
+
+pbjs.que.push(function() {
+
+    pbjs.setConfig({
+        s2sConfig: {
+            accountId: '1',            
+            bidders: ['appnexus', 'pubmatic'],
+            defaultVendor: 'appnexus'
```

Comment:
I think this block needs the k-v `enabled: true` according to the [Prebid Server bidder params](http://prebid.org/dev-docs/bidders.html#prebidServer)

Reply:
Ah, ok!